src/js/parsers/captions/dfxp.js

```diff
@@ -10,6 +10,8 @@ define([
         validate(xmlDoc);
         var _captions = [];
         var paragraphs = xmlDoc.getElementsByTagName('p');
+        var tt = xmlDoc.getElementsByTagName('tt');
+        var frameRate = (tt && tt[0].getAttribute) ? tt[0].getAttribute('ttp:frameRate') || 30 : 30;
```

Comment:
Simplify this. It's better to use an if statement than OR in a ternary statement.